homework_4/task2.py

```diff
@@ -0,0 +1,13 @@
+import random
+
+n = int(input("Enter Number from 1 to 30: "))
+random_numbers = [] 
+
+for _ in range(n):
+    random_number = random.randint(0, 1000)
+    random_numbers += [random_number] 
```

Comment:
ეს დავალება ლისტის გარეშეც შეიძლებოდა:
from random import randint
n = int(input('Enter a whole number n so that 0 < n < 30: '))

if not 0 < n < 30:
    print('Entered number is not between 0 and 30.')
    exit(1)

max_n = 0
for i in range(n):
    random_number = randint(0, 1000)
    if random_number > max_n:
        max_n = random_number

print(max_n)